src/app/services/api.ts

```diff
@@ -303,6 +303,9 @@ export const api = {
     testFreeTextQuestion: (userDefinedChoices: Choice[], testCases: TestCaseDTO[]) => {
       return endpoint.post("/questions/test?type=isaacFreeTextQuestion", { userDefinedChoices, testCases });
     },
+    randomQuestions: (): AxiosPromise<ApiTypes.IsaacQuestionPageDTO[]> => {
+      return endpoint.get("/questions/randomQuestions");
```

Comment:
This is more of an API comment, really, but it should be `random_questions`, since it seems like we tend to use underscores for URLs, rather than camel case.

Reply:
Agreed, happy to update this when the endpoint is renamed in API.

Reply:
renamed endpoint to `/questions/random` as per API changes